webclient/static/js/arch_z_detail_map.js

```diff
@@ -1191,9 +1280,17 @@ function checkBlockedByQWuery(){
 }
 
 window.addEventListener("load", function(){
-    loadSession();
+    if( global_map_can_edit==true) {
+        loadSession();
+        geomToText();
+        save_edited_geometry_session()
+    } 
+    else{
+        sessionStorage.removeItem("Geom-session");
+    }
 });
 
+//kontrola geometrie asi nefunguje
```

Comment:
@jhavrlant Pokud nefunguje, chtělo by opravit. Nebo je to zastaralý koment?

Reply:
Ta poznámka není úplně přesná. Ta funkce má kontrolovat importovanou geometrii v GET, což dělá i když je tam chyba, ale raději bych ji vůbec nepoužil a opravil přes #1369, protože mi přijde neefektivní kontrolovat geometrii dvakrát za sebou a pak ještě při ukládání. Také se neukáže žádná varovná hláška.

Reply:
Ok, tak vyřešme v rámci toho #1369, jak Ti to přijde nejlepší.